superset/config.py

```diff
@@ -203,6 +204,9 @@
 # Maximum number of rows returned in the SQL editor
 SQL_MAX_ROW = 1000
 
+# Maximum number of tables/views displayed in the dropdown window in SQL Lab.
```

Comment:
wondering about the rationale for using 3000 here. is the dropdown still performant with this many records?